src/Features/Core/Portable/UseThrowExpression/AbstractUseThrowExpressionDiagnosticAnalyzer.cs

```diff
@@ -62,7 +62,7 @@ protected override void InitializeWorker(AnalysisContext context)
                 s_registerOperationActionInfo.Invoke(startContext, new object[]
                 {
                     new Action<OperationAnalysisContext>(operationContext => AnalyzeOperation(operationContext, expressionTypeOpt)),
-                    ImmutableArray.Create(OperationKind.ThrowStatement)
+                    ImmutableArray.Create(OperationKind.ExpressionStatement)
```

Comment:
I'd prefer to register .ThrowExpressoin, then only check ThrowExpressions that are contianed in ExpressionStatements.  that's because ThrowExpressions will be much rarer than expression statements.